src/components/DagComponent.js

```diff
@@ -365,6 +365,51 @@ class DagComponent extends React.Component {
       });
   }
 
+  newDrawGraph() {
```

Comment:
question: Why did we need to re-implement this method? It was dropped on #345

Reply:
You are right, it has been corrected. Please review.